NovelRT.Sdk/NovelRT.Sdk.csproj

```diff
@@ -13,6 +13,9 @@
       <None Update="TemplateFiles\CMakeTemplate\cmake\CopyDirectoryRaw.cmake">
         <CopyToOutputDirectory>Always</CopyToOutputDirectory>
       </None>
+      <None Update="TemplateFiles\CMakeTemplate\Resources\DeleteMe.txt">

```

Comment:
This was found testing the generator - with the CMakeLists file in the folder, the Resources folder wouldn't generate. However, if we copy using the `copy_directory_raw` function, adding the DeleteMe.txt in its place allows your code to clean the folder up as expected.